build.gradle

```diff
@@ -13,7 +13,9 @@ buildscript {
 	    url = "https://repo1.maven.org/maven2/"
     }
     dependencies {
-        classpath 'net.minecraftforge.gradle:ForgeGradle:1.2-SNAPSHOT'
+        classpath ('com.anatawa12.forge:ForgeGradle:1.2-1.0.+') {
```

Comment:
Haven't heard much about this, any difference to ForgeGradle besides getting rid of old warnings?

Reply:
FG now refuses to build without cached stuff - some of the sources have been moved around.

https://stackoverflow.com/questions/74504545/forge-1-7-10-error-while-gradlew-setupdecompworkspace

Willy sent me the fixed build.gradle

Reply:
Ah, good to know.